servers/src/proto/journal/journal.proto

```diff
@@ -17,6 +18,9 @@ message JournalEntry {
     BlockContainerIdGeneratorEntry block_container_id_generator = 3;
     BlockInfoEntry block_info = 4;
     CompleteFileEntry complete_file = 5;
+    CompletePartitionEntry complete_partition = 22;
```

Comment:
good call. the answer is maybe, but not in this PR (or in this release), as we could rename the proto field when we see a naming conflict without breaking backward compatibility.